app/repositories/app_usage_event_repository.rb

```diff
@@ -110,6 +110,8 @@ def purge_and_reseed_started_apps!
           space_name:                         "#{Space.table_name}__name".to_sym,
           org_guid:                           "#{Organization.table_name}__guid".to_sym,
           created_at:                         Sequel.datetime_class.now,
+          parent_app_name:                    :parent_app__name,
+          parent_app_guid:                    "#{ProcessModel.table_name}__guid".to_sym,
```

Comment:
should this be `#{AppModel.table_name}__guid`?

Reply:
I don't think the reference to AppModel resolves (at least it didn't when I tried). However, it looks like we can just reference parent_app__guid directly. I updated the PR and also added a test to confirm parent_app_guid and app_guid end up being equal.

Reply:
One last change -- I updated the test to show that `parent_app_name` and `parent_app_guid` are set to the values of the App name and guid.